docs/index.html

```diff
@@ -105,104 +104,67 @@ <h1 class="au-header__heading">Example Site</h1>
+					<a class="au-btn" href="https://designsystem.gov.au/components/buttons/">
+						Read about buttons
+					</a>
 				</div>
-				<div class="col-sm-6">
-					<img class="au-responsive-media-img banner__image" src="http://placehold.it/900x300" alt="">
+				<div class="col-sm-12 col-md-6">
+					<img class="au-responsive-media-img banner__image" src="http://placehold.it/900x400" alt="">
 				</div>
 			</div>
 		</div>
 	</div>
 
-	<div class="container-fluid au-body main-content">
-		<!-- BREADCRUMBS -->
-		<div class="row">
-			<div class="col-md-12">
-				<nav class="au-breadcrumbs" aria-label="breadcrumb">
-					<ul class="au-link-list au-link-list--inline">
-						<li>
-							<a href="#">Home</a>
-						</li>
-						<li>
-							<a href="#">Responsibilities</a>
-						</li>
-						<li>
-							<a href="#">User needs</a>
-						</li>
-						<li>
-								<a href="#">Understand user needs</a>
-						</li>
-						<li>Community</li>
-					</ul>
-				</nav>
-			</div>
-		</div>
-
-		<div class="row">
-			<!-- SIDE NAV -->
-			<div class="col-sm-4 col-md-3">
-				<aside id="nav" class="au-side-nav au-accordion">
-					<a href="#nav-default" class="au-side-nav__toggle au-accordion__title au-accordion--closed" aria-controls="nav-default" aria-expanded="false"
-					 aria-selected="false" role="tab" onclick="return AU.accordion.Toggle( this )">
-						In this section
-					</a>
-					<div id="nav-default" class="au-side-nav__content au-accordion--closed au-accordion__body" aria-hidden="true">
-						<h2 class="au-sidenav__title">
-							<a href="#">Responsibilities</a>
-						</h2>
-						<ul class="au-link-list">
-							<li>
-								<a href="#">Side-nav item level 1</a>
-							</li>
-							<li>
-								<a href="#">User Needs</a>
-								<ul class="au-link-list">
-									<li>
-										<a href="#">Understand user needs</a>
-										<ul class="au-link-list">
-											<li class="active">
-												<a href="#">Community</a>
-											</li>
-										</ul>
-									</li>
-								</ul>
-							</li>
-							<li>
-								<a href="#">Side-nav item level 1</a>
-							</li>
-						</ul>
+	<!-- MAIN BODY -->
+	<main class="container-fluid au-body main-content">
+		<div class="container-fluid">
+			<div class="row">
+				<!-- The 2 classes below col-sm-6 and col-md-4 set the width of the section based on the viewport width -->
+				<section class="col-sm-6 col-md-4">
+					<div class="content-card">
+						<img class="au-responsive-media-img" src="//via.placeholder.com/350x200" alt="Placeholder image description">
+						<h3>Use the grid system</h3>
+						<p>The <code>col-md-4</code> class (the md stands for medium, the 4 means that this column will take up 4 of 12 grid spaces) on this <code>&lt;section&gt;</code> tag turns it into one of 3 columns on a screen greater than 992px. You will see the other class <code>col-sm-6</code> changes the width of the section based on the size of the screen the website is being viewed on.</p>
```

Comment:
with the `<p>` inside the cards do you think it would be better to justify the text? Looks cleaner in my opinion. Adding following CSS to the `<p>` 

`text-align: justify`

we get:

<img src="https://user-images.githubusercontent.com/20184809/46057616-d5726080-c199-11e8-8b37-9e27ff383cda.png" width="700" />


instead of

<img src="https://user-images.githubusercontent.com/20184809/46057647-15d1de80-c19a-11e8-8dd9-64da7eb5c041.png" width="700" />

Reply:
also maybe it's me,  but do the columns feel a little too close together?